shell/browser/net/proxying_url_loader_factory.cc

```diff
@@ -810,6 +810,7 @@ void ProxyingURLLoaderFactory::CreateLoaderAndStart(
   network::ResourceRequest request = original_request;
 
   if (ShouldIgnoreConnectionsLimit(request)) {
+    request.priority = net::RequestPriority::MAXIMUM_PRIORITY;
```

Comment:
This seems unrelated to the PR title, is it intentionally here?

Reply:
I can move this into its own PR.